kernel/time/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "time"
+version = "0.1.0"
+authors = ["Klim Tsoutsman <klim@tsoutsman.com>"]
+description = "Abstraction for hardware timers"
```

Comment:
see my top-level comment in which I recommend distinguishing between a clock(source) and a timer